### PR TITLE
[keycard] [e2e test menu] Regular multiacc as a backend for simulated keycard

### PR DESCRIPTION
**Pull request by rasom**
This change clears path for sending messages (should work already) and signing txs with simulated keycard. 
Also it allows to log into simulated kk multiacc and restore such account with mnemonic.
Signing will be implemented separately.

status: ready

### CODE REVIEW

**Review by churik — APPROVED** [review]

**Comment by churik** [comment]
@rasom please mind my commit while squashing

**Review by Serhy — APPROVED** [review]

File: src/status_im/utils/handlers.cljs [hunk 1]
```diff
@@ -82,8 +82,8 @@
 
 (def default-interceptors
   [debug-handlers-names
-   (when js/goog.DEBUG
-     check-spec)
+   (when false #_js/goog.DEBUG
```
Comment (flexsurfer) [inline src/status_im/utils/handlers.cljs]: RIP spec

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/utils/handlers.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/utils/handlers.cljs]: I will add it back in this PR, but do you plan to remove it?

Comment (flexsurfer) [reply src/status_im/utils/handlers.cljs]: yes soon

**Comment by churik** [comment]
Failed e2e is not related to changes in PR.
ready to merge

**Pull request merged**